connectors/src/connectors/google_drive/temporal/activities.ts

```diff
@@ -847,6 +847,15 @@ async function deleteFile(googleDriveFile: GoogleDriveFiles) {
     const dataSourceConfig = dataSourceConfigFromConnector(connector);
     await deleteFromDataSource(dataSourceConfig, googleDriveFile.dustFileId);
   }
+  const folder = await GoogleDriveFolders.findOne({
+    where: {
+      connectorId: connectorId,
+      folderId: googleDriveFile.driveFileId,
+    },
+  });
+  if (folder) {
```

Comment:
is it worth making a transaction here? intuition: no, too edgy
but open to discussion

Reply:
I would indeed delete the GoogleDriveFolder and GoogleDriveFile in one transaction.